src/ChargeTrapping/ChargeTrapping.jl

```diff
@@ -121,3 +121,85 @@ end
 
 
 
+
+"""
+    struct ConstantLifetimeChargeTrappingModel{T <: SSDFloat} <: AbstractChargeTrappingModel{T}
+        
+This constant-lifetime-based charge trapping model is similar to the Boggs model, which is constant-mean-free-path based.
+
+The model implemented has two sets of parameters for the sensitive (bulk) and inactive (dead layer/surface layer) volume respectively.
+The inactive layer effect will only be considered if the corresponding `virtual_drift_volume` is defined in the configuration file.
+
+## Fields
+* `τh::T`: Lifetime for holes in bulk volume (sensitive region) (default: `τh = 1ms`).
+* `τe::T`: Lifetime for electrons in bulk volume (default: `τe = 1ms`).
+* `τh_inactive::T`: Lifetime for holes in surface layer (dead layer/inactive layer) (default: `τh_inactive = 1ms`).
+* `τe_inactive::T`: Lifetime for electrons in surface layer (default: `τe_inactive = 1ms`).
+* `inactive_layer_geometry`: The geometry of the inactive layer. When this is defined, the model will consider the inactive effect.
+> Note: All τ must be much bigger than `Δt`.
+
+See also [Charge Trapping Models](@ref).
+"""
+struct ConstantLifetimeChargeTrappingModel{T <: SSDFloat} <: AbstractChargeTrappingModel{T} 
+    τh::T
+    τe::T
+    τh_inactive::T
+    τe_inactive::T
+    inactive_layer_geometry
+end
```

Comment:
Check if this works. You should give `inactive_layer_geometry` a type and add it as parametric type to the struct.
```suggestion
struct ConstantLifetimeChargeTrappingModel{T <: SSDFloat, G <: AbstractGeometry} <: AbstractChargeTrappingModel{T} 
    τh::T
    τe::T
    τh_inactive::T
    τe_inactive::T
    inactive_layer_geometry::G
end
```
if it can also be missing, try something like:
```suggestion
struct ConstantLifetimeChargeTrappingModel{T <: SSDFloat, G <: Union{<:AbstractGeometry, Missing} <: AbstractChargeTrappingModel{T} 
    τh::T
    τe::T
    τh_inactive::T
    τe_inactive::T
    inactive_layer_geometry::G
end
```
Maybe also consider `nothing` instead of `missing` here, if nothing is passed.

Reply:
Careful: here you would need a capital N because it’s a type, not a variable! 

nothing => Nothing